src/DropdownMenu.tsx

```diff
@@ -102,6 +143,21 @@ const DropdownMenu: DropdownMenu = React.forwardRef(
     const prefix = useBootstrapPrefix(bsPrefix, 'dropdown-menu');
     const [popperRef, marginModifiers] = usePopperMarginModifiers();
 
+    const alignClasses: string[] = [];
+    if (align) {
+      if (typeof align === 'object') {
+        DEVICE_SIZES.forEach((brkPoint) => {
+          const direction = align[brkPoint];
+          if (direction) {
+            alignRight = alignRight || direction === 'left';
```

Comment:
**question:** i'm confused by this – we set `alignRight` to `true` if _any_ `direction` is `left`? that seems a little odd to me

Reply:
Sorry should've left a comment here.  In order to responsively align left, the `dropdown-menu-right` class needs to be added in addition to `dropdown-menu-**-left`.  I reused this bool to control toggling of that class.

https://getbootstrap.com/docs/4.5/components/dropdowns/#responsive-alignment